cognite/neat/_rules/models/entities/_wrapped.py

```diff
@@ -42,8 +41,8 @@ def _parse(cls, data: str) -> dict:
 
         # raw filter case:
         if cls.__name__ == "RawFilter":
-            if match := re.search(r"rawFilter\(([\s\S]*?)\)", data):
-                return {"filter": match.group(1), "inner": None}
+            if data.startswith("rawFilter(") and data.endswith(")"):
+                return {"filter": data[10:-1], "inner": None}
```

Comment:
this slice here is very specific. Any possiblity of having out-of-bounds errors here when doing this? Looks a bit dangerous, but I lack a bit on the context here